### DISCUSSION TITLE
On specifying the spin occupations of the quantum chemistry problem.

### DISCUSSION THREAD

**Discussion by jiangtong1000** [shuaigroup/Renormalizer#160] [General]
Reno previously specified the total electron number `qntot`, for constructing the quantum chemistry model. The model then explored the ground state within the confines of this `qntot` space. Now, it requires the specification of the `qn` for spin-up and spin-down electrons.

A question arises: in the case of a given system, it's not common to have prior knowledge about the spin configurations of its ground state. Does the ground state has to be half up, half down? No, right? I was thinking if the original feature should be retained.

**Comment by liwt31** [comment]
Hi Dr. Jiang. I believe that currently there's no obstacle to specifying only one quantum number. in `qc_model`, two quantum numbers are assumed. Maybe an option can be added to the `conserve_qn` argument for the case that only the total number of electrons is conserved.

On the other hand, I think it is quite *common to have prior knowledge about the spin configurations of its ground state*. Just think of a PySCF or Gaussian calculation, you always have to specify the spin configuration, right?

Cheers

**Reply by jiangtong1000** [reply]
Oh no, my secret identity has been revealed. No need for the formal title next time lol.
